web_client/src/actions/auth.js

```diff
@@ -0,0 +1,65 @@
+import { push } from 'connected-react-router';
+import { fetchRefreshToken, fetchPlaylists } from 'services/api';
+import {
+  fetchPlaylistsStarted,
+  fetchPlaylistsSucceeded
+} from 'actions/playlists';
+
+export const FETCH_REFRESH_TOKEN_SUCCEEDED = 'FETCH_REFRESH_TOKEN_SUCCEEDED';
+export const FETCH_ACCESS_TOKEN_SUCCEEDED = 'FETCH_ACCESS_TOKEN_SUCCEEDED';
+export const FETCH_REFRESH_TOKEN_STARTED = 'FETCH_REFRESH_TOKEN_STARTED';
+export const FETCH_ACCESS_TOKEN_STARTED = 'FETCH_ACCESS_TOKEN_STARTED';
+export const USER_LOGGED_OUT = 'USER_LOGGED_OUT';
+
+export const fetchRefreshTokenSucceeded = (refreshToken) => ({
+  type: FETCH_REFRESH_TOKEN_SUCCEEDED,
+  payload: {
```

Comment:
looks like this isn't relevant anymore, but airbnb's styleguide uses spaces around equal sign in default params:

https://github.com/airbnb/javascript#es6-default-parameters

Reply:
I'm guessing this is in reference to the default param spacing comment?

Reply:
yeah. sorry for night carrying over the context.